src/firmware/README.md

```diff
@@ -1,12 +1,34 @@
 # Fimware
 
-### gps_firmware
+### Udev Rules
 
-- Designed to allow pass lat and lon coordinates from a [Adafruit Ultimate GPS Breakout v3](https://www.adafruit.com/product/746) to a PC over serial
-- Sends messages in the form `GPS,lat,lon,has_gps_fix`, where `has_gps_fix` is a 1 or 0 representing whether or not the GPS has a fix
+Udev assigns the names specified in the rule file to the devices, which helps matching dev files with devices.
 
-char to send | what it does
--------------| ------------
-`d` | toggles debug mode. in debug mode, the gps will send messages over serial as it receives them
-`r` | requests the most recent gps message (it will be returned over serial)
+- Rule Writing Guideline
+	- Create file: `/etc/udev/rules.d/10-snowbots.rules` (where xx represent the priority; Here 10 is higher than 50, the default)
```

Comment:
Since we've already created the file, can we remove this step?

Reply:
Right. I'm wondering if this line can still be kept with a comment below saying this step is already done automatically in the first time setup. Creating the rule file is the first step of the udev rules setup. The complete udev rules writing guideline allows other members to make some changes in the bash file for the udev rules setup later.

Reply:
I think that if the bash file is sufficiently commented, this should be enough documentation for people to understand how we're creating the udev rules. The README should just detail what people will need to know in order to write and update them.

Reply:
Ok, I'll delete the first step and push it again.